flatpak_builder_lint/staticfiles/exceptions.json

```diff
@@ -1118,6 +1118,9 @@
     "org.kde.kdevelop": {
 	"finish-args-flatpak-spawn-access": "the app predates this linter rule"
     },
+    "org.libretro.RetroArch": {
+        "finish-args-contains-both-x11-and-wayland": "fallbacks to wayland on non-x11 system"
```

Comment:
I agree it's pretty poor. I had copied the description from Thunderbird's below this entry.

> fallbacks to wayland on non-x11 system

RetroArch has support for both wayland and x11. How about....

``` suggestion
        "finish-args-contains-both-x11-and-wayland": "falls back to x11 when wayland is not available"
```

Reply:
then you don't need the exception and just have `fallback-x11` instead of `x11` as the linter rule is about.

About t-bird, also had the same comment but it was merged anyway.

Reply:
Ah, didn't know of fallback-x11. Thanks so much :+1:

Reply:
@hfiguiere t-bird (or every mozilla app) is unique because it uses wayland as secondary choice (not primary as most apps do) only when no x11 socket (xwayland) is available. fallback-x11 artificially creates such environment but the developers intent is to use x11 even on wayland environment unless there is no choice.